KarhooUISDK/CommonUI/BottomSheet/KarhooBottomSheetViewModel.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  BottomSheetMPV.swift
+//  KarhooUISDK
+//
+//  Created by Diana Petrea on 03.11.2022.
+//  Copyright © 2022 Flit Technologies Ltd. All rights reserved.
+//
+
+import Foundation
+import SwiftUI
+
+protocol BottomSheetViewModel {
+    var title: String { get set }
+    var cornerRadius: CGFloat { get set }
+    var backgroundColor: UIColor { get set }
+    var callback: ScreenResultCallback<AnyObject>? { get set }
+}
+
+class KarhooBottomSheetViewModel: BottomSheetViewModel {
+    var title: String = "Some Title"
```

Comment:
Are you sure we need debug title?

Reply:
No, it was there for the testing demo I added to the ticket and forgot about it during cleanup. I'll remove the value.